README.md

```diff
@@ -8,6 +8,11 @@ Ruby Application for the Vroegtijdig School Verlaten Dagboekonderzoek
 ## Reference
 Emerencia, A.C., Blaauw, F.J., Snell, N.R., Blijlevens, T., Kunnen, E.S., De Jonge, P. & Van der Gaag, M.A.E. (2017). U-can-act Web-app (Version 1.0) [Web application software]. Retrieved from [www.u-can-act.nl](www.u-can-act.nl)
 
+## Funding
+This application (and the accompanying research project) is powered by funding from The Netherlands Initiative for Education Research (NRO) under projectnumber 405-16-401.
```

Comment:
cool mn eerste pullrequest review :)
Tnx. Hier nog een suggestie voor een iets letterlijkere vertaling voor het eerste deel van de zin, want ik geloof dat ze waarde hechten aan precieze bewoording: 'This application has been made possible by funding from ....'

Reply:
:+1: